src/app.ts

```diff
@@ -48,9 +48,11 @@ controller.registerCommand(commandList.controller.help, controller.onHelp.bind(c
 
 // Botty commands
 controller.registerCommand(commandList.botty.restart, bot.onRestart.bind(bot));
+controller.registerCommand(commandList.botty.version, bot.onVersion.bind(bot));
 
 // API schema commands
 controller.registerCommand(commandList.apiSchema.updateSchema, apiSchema.onUpdateSchemaRequest.bind(apiSchema));
+controller.registerCommand(commandList.apiUrlInterpreter.updateSchema, apiUrlInterpreter.onUpdateSchemaRequest.bind(apiUrlInterpreter));
```

Comment:
not sure what line is the correct one atm, it appeared as a merge issue so i just added both :LUL: